### DISCUSSION TITLE
プロフィールアイコンと、リアクションのスタンプの作成

### DISCUSSION THREAD

**Discussion by tkg-geek** [TeamC-by-TNI/Felizo#16] [General]
打ち合わせで話したプロフィールアイコンとリアクションスタンプについての共有です。
各自好きに制作をして、22日のMTGで共有しましょう。凝りすぎなくて結構です。

#### プロフィールアイコン（1人3点）
---
以下3点を書いてください。
・動物的なキャラクター
・空想のUMA（未確認生物）
・自由に何かしらのオリジナルキャラクター

作り方
・手書き（白紙、罫線のない紙ならOK）
・デジタル（白背景であれば何でもOK）
※書いたものを統一したデジタルデータに変換します。

#### リアクションスタンプ
---
Slackの絵文字スタンプのイメージでFelizoで使うリアクションスタンプを自由に3点（3点以上OK）を制作ください。

作り方
以下の2サイトで128x128サイズのスタンプが作れます。
https://zk-phi.github.io/MEGAMOJI/
https://emoji-gen.ninja/
こちらで制作ください。

参考に、最近始まったmixi2のスタンプを添付します。
![324538_0](https://github.com/user-attachments/assets/37becf08-fc42-41fa-abce-f8b1c88246b8)
![324539_0](https://github.com/user-attachments/assets/1172758b-14e1-45f8-b038-73207e707155)

**Comment by mikiIDE** [comment]
有難う～助かります！
稼働スケジュールを参考に、
22日（日）の定期ミーティングは**10:30～の朝のみ**という認識でいいですか？👀

**Reply by tkg-geek** [reply]
そうだね、その認識です！
※僕は終日空いているのでもし必要になれば（みんなの都合がつけば）ミーティングできます。

**Reply by mikiIDE** [reply]
↑変更があったので念のため書き込み！
日曜のMTGは20:00～です！

**Comment by mikiIDE** [comment]
こちらは提出が終わったためクローズします！
